src/nise/metadata/config.py

```diff
@@ -0,0 +1,26 @@
+config = {
+    'metadata_version': '0.9',
+    'module_name': 'nise',
+    'module_version': '0.9.0.dev0',
+    'c_function_prefix': 'niSE_',
+    'driver_name': 'NI Switch Executive',
+    'session_class_description': 'An NI Switch Executive session',
+    'session_handle_parameter_name': 'vi',
+    'library_info':
+    {
+        'Windows': {
```

Comment:
By not having Linux entries, users may get a misleading error. In other drivers we put something there even though the driver doesn't support Linux (i.e. nifgen). Please test it out at least.

Reply:
fixed